src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -64,6 +65,7 @@ const ReimbursementRequestTable = ({
                 <TableCell align="center">{dateUndefinedPipe(row.dateSubmittedToSabo)}</TableCell>
                 <TableCell align="center">{row.vendor.name}</TableCell>
                 <TableCell align="center">{cleanReimbursementRequestStatus(row.status)}</TableCell>
+                <TableCell align="center">{row.refundSource}</TableCell>
```

Comment:
We should create a pipe (there might already be one) to lowercase the rest of the string (Budget instead of BUDGET).
See cleanReimbursementRequestStatus for an example.

Reply:
I just resolved this issue but idk how to merge my branch. I tried to commit it under my local #2079 branch but when I try to make a PR my branch won't show up.

Reply:
I think there were already changes made by anya for the branch. Thus when you go to make changes make sure to pull the latest changes from the branch. 
When you're on the desired branch (use **git branch** to verify) run **git pull** to pull the latest changes from the branch.